fplus-http-server/src/router/application.rs

```diff
@@ -456,3 +456,13 @@ pub async fn check_for_changes(
 pub async fn health() -> impl Responder {
     HttpResponse::Ok().body("OK")
 }
+
+#[post("application/trigger_datacap_request")]
+pub async fn trigger_datacap_request(info: web::Json<TriggerDataCapRefill>) -> impl Responder {
```

Comment:
```suggestion
pub async fn trigger_ssa(info: web::Json<TriggerSSAInfo>) -> impl Responder {
```
`Info`, cause that's the convention used in all other POST requests in this app.
`TriggerSSA` to match endpoint